develop/tutorials/articles/100-tooling/04-liferay-workspace/05-managing-the-target-platform-for-liferay-workspace.markdown

```diff
@@ -0,0 +1,153 @@
+    }
+
++$$$
+
+**Note**: The `liferay.workspace.target.platform.version` property also sets
+the distro JAR used in workspace to validate your projects during the build
+process. See the
+[Validating Modules Against the Target Platform](/develop/tutorials/-/knowledge_base/7-1/validating-projects-against-the-target-platform)
+tutorial for more info.
+
+$$$
+
+The target platform functionality is available in Liferay Workspace version
+1.8.1+. If you have an older version, you must update it to leverage platform
+targeting. See the
+[Updating Liferay Workspace](/develop/tutorials/-/knowledge_base/7-1/updating-liferay-workspace)
+tutorial to do this.
+
+<!-- TODO: Verify that the 1.8.1 version is accurate once the features are
+officially released. -Cody -->
+
+You now know how to configure a target platform in workspace and how
+dependencies without versions appear in your Gradle build files. You're all set!
+
+## Targeting a Platform Outside of Workspace
+
+<!-- TODO: Remove this section once the reference doc for the Gradle plugin is
+available. This is just gives a taste of targeting a platform for those who
+don't want to use Workspace. More details will be provided in reference doc.
+-Cody -->
+
+If you prefer to not use Liferay Workspace, but still want to target a platform
+to develop against, you must apply the
+[Target Platform](https://github.com/liferay/liferay-portal/tree/master/modules/sdk/gradle-plugins-target-platform)
+Gradle plugin to the root `build.gradle` file of your custom multi-module Gradle
+build.
+
+To do this, your `build.gradle` file should look similar to this:
+
+    buildscript {
+        dependencies {
+            classpath group: "com.liferay", name: "com.lifereay.gradle.plugins.target.platform", version "1.0.0"
+        }
+        repositories {
+            maven {
+                url "https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public"
+            }
+            maven {
+                url "https://repository.lifray.com/nexus/service/local/repositories/liferay-public-snapshots/content"
+            }
+            mavenLocal()
+        }
+    }
+
+    apply plugin: "com.liferay.target.platform"
+
+    dependencies {
+        targetPlatformBoms group: "com.liferay", name: "com.liferay.ce.portal.bom", version: "7.0.5-SNAPSHOT"
+        targetPlatformBoms group: "com.liferay", name: "com.liferay.ce.portal.compile.only", version: "7.0.5-SNAPSHOT"
+    }
+
+This Gradle code
+
+- applies Liferay's Target Platform Gradle plugin
+- configures repositories that will provide the necessary artifacts for your
+  project build
+- sets the Target Platform plugin's dependencies:
+    - `com.liferay.ce.portal.bom`: provides all the artifacts included in
+      @product@.
+    - `com.liferay.ce.portal.compile.only`: provides artifacts that are not
+      included in @product@, but are necessary to reference during the build
+      (e.g., `org.osgi.core`).
+
+Now you can develop against a target platform!
```

Comment:
we need to add some content about the `targetPlatformIDE { includeGroups }` extension as a way to control what is available in the 'target platform index'